src/Data/Conversion.hs

```diff
@@ -164,6 +164,12 @@ instance Conversion Text String where
 instance Conversion String Text where
   convert = Text.unpack
 
+class Conversion Text a => ToText a where
+  toText :: a -> Text
+
+instance Conversion Text a => ToText a where
+  toText = convert
```

Comment:
Why not have this as a default instance such that users can be free to define more custom instance if desired.

Reply:
good call.

But also your comment lets me think about something else: 

Why not:

```
type ToText a = Conversion Text a
```

And keep the original `toText`.

We do never want the `convert` and `toText` to differ IMO. Hence making a subclass which makes this possible is a bad idea.

`toText / ToText` should just be an alias.

Reply:
> We do never want the convert and toText to differ IMO.

I like this rationale. Let's go fro an alias.